static/app/components/modals/inviteMembersModal/index.spec.tsx

```diff
@@ -159,120 +187,154 @@ describe('InviteMembersModal', function () {
   });
 
   it('can be closed', async function () {
-    jest.mocked(useOrganization).mockReturnValue(org);
     const close = jest.fn();
+    const modalProps = {
+      ...defaultMockModalProps,
+      closeModal: close,
+    };
+    setupView({modalProps});
 
-    render(<InviteMembersModal {...modalProps} closeModal={close} />);
-
-    await waitFor(() => {
-      expect(screen.getByRole('button', {name: 'Cancel'})).toBeInTheDocument();
-    });
+    expect(await screen.findByRole('button', {name: 'Cancel'})).toBeInTheDocument();
 
     await userEvent.click(screen.getByRole('button', {name: 'Cancel'}));
     expect(close).toHaveBeenCalled();
   });
 
-  it('sends all successful invites', async function () {
-    jest.mocked(useOrganization).mockReturnValue(org);
-    const createMemberMock = MockApiClient.addMockResponse({
-      url: `/organizations/${org.slug}/members/`,
-      method: 'POST',
+  it('sends all successful invites without team defaults', async function () {
+    const {mocks} = setupView({
+      mockApiResponses: [defaultMockOrganizationRoles, defaultMockPostOrganizationMember],
     });
 
-    render(<InviteMembersModal {...modalProps} />);
-
-    await waitFor(() => {
-      expect(screen.getByRole('button', {name: 'Add another'})).toBeInTheDocument();
-    });
+    expect(await screen.findByRole('button', {name: 'Add another'})).toBeInTheDocument();
 
-    // Setup two rows, one email each, the first with a admin role.
-    await userEvent.click(screen.getByRole('button', {name: 'Add another'}));
-
-    const emailInputs = screen.getAllByRole('textbox', {name: 'Email Addresses'});
-    const roleInputs = screen.getAllByRole('textbox', {name: 'Role'});
+    await setupMemberInviteState();
     const teamInputs = screen.getAllByRole('textbox', {name: 'Add to Team'});
-
-    await userEvent.type(emailInputs[0], 'test1@test.com');
-    await userEvent.tab();
-    await selectEvent.select(roleInputs[0], 'Admin');
     await selectEvent.select(teamInputs[0], '#team-slug');
-
-    await userEvent.type(emailInputs[1], 'test2@test.com');
-    await userEvent.tab();
+    await selectEvent.select(teamInputs[1], '#team-slug');
```

Comment:
not sure why this is failing to "unselect" the team and remove it from the option now after the test refactor